docs/source-pytorch/common/checkpointing_basic.rst

```diff
@@ -122,6 +122,20 @@ If you used the *self.save_hyperparameters()* method in the init of the Lightnin
     # uses in_dim=128, out_dim=10
     model = LitModel.load_from_checkpoint(PATH, in_dim=128, out_dim=10)
 
+Similarly, if you used other positional arguments or keyword arguments in the ``__init__`` method of the LightningModule,
+you can initialize the model as follows:
```

Comment:
There is still one missing concept here that we didn't explain, which I think would resolve some confusion.
We've explained that

1) if you call `save_hyperparameters()`, the init params get saved and then when you load using `load_from_checkpoint`, you can override them if you want but it's optional.


But we haven't explained

2) If you didn't call `save_hyperparameters()` OR you excluded some parameters to be saved via `save_hyperparameters(ignore=...)` then you MUST pass the missing arguments when calling `load_from_checkpoint`. This is precisely what was needed in the example that @rasbt discovered. 

So I would use this opportunity to extend the sentence above to explain that.

Reply:
Thanks @awaelchli . I extended the explanation a bit to clarify.